scripts/user-operation.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Script to execute an ERC-4337 user operation for a test executor.
+ */
+
+import { ethers, deployments } from "hardhat";
+
+import { PLUGIN_PERMISSION_EXECUTE_CALL } from "../src/utils/constants";
+import { UserOperationStruct } from "../typechain-types/@account-abstraction/contracts/interfaces/IAccount";
+
+const BUNDLER = process.env.SAFE_PROTOCOL_ERC4337_BUNDLER_URL ?? "http://localhost:3000/rpc";
+const ENTRYPOINT = process.env.SAFE_PROTOCOL_ERC4337_ENTRYPOINT ?? "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789";
+
+async function main() {
+    const bundler = await bundlerRpc(BUNDLER);
+
+    const manager = await deployments
+        .get("SafeProtocolManager")
+        .then(({ address }) => ethers.getContractAt("SafeProtocolManager", address));
+    const module = await deployments
+        .get("SafeProtocol4337Module")
+        .then(({ address }) => ethers.getContractAt("SafeProtocol4337Module", address));
+    const handler = await ethers.getContractAt("ISafeProtocol4337Handler", await module.getAddress());
+    const entrypoint = await ethers.getContractAt("EntryPoint", ENTRYPOINT);
+
+    const account = await ethers.deployContract("TestExecutor", [await manager.getAddress()]);
```

Comment:
Would it make sense to use an actual safe account here?

Reply:
Probably, but it made the signing more complicated 😅. You _could_ argue that the PoC was about ensuring that some account implementation paired with the Safe{Core} Protocol reference implementation could work with ERC-4337, so I don’t think its strictly necessary.